governance.md

```diff
@@ -109,6 +110,8 @@ This is the purpose of Working Groups (WG). The intent is to make
 Working Groups relatively easy to create and to deprecate, once
 inactive.
 
+Working groups do not own any code or subprojects. Instead, they are a place for people to discuss topics that cross SIG boundaries.
```

Comment:
Do you think it would be worth adding a clarification that you do not need a working group to collaborate with others in the community? This seems to be a common theme in the various WG/SIG proposal discussions going on right now.

Reply:
Good idea -- added in the next revision.